app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
```suggestion
  belongs_to :user`, class_name: 'User'`
```

- [ ] It would be nice to add the class name User to your association setup above

Reply:
This change has been implemented well